test/suites/smoke/test-old-regressions.ts

```diff
@@ -0,0 +1,110 @@
+import "@moonbeam-network/api-augment";
+import { describeSuite, beforeAll, expect } from "@moonwall/cli";
+import { ApiPromise } from "@polkadot/api";
+import { ethers } from "ethers";
+import { encodeFunctionData } from "viem";
+
+describeSuite({
+  id: "S15",
+  title: "Verify regressions which happened in the past",
+  foundationMethods: "read_only",
+  testCases: async ({ context, it, log }) => {
+    let paraApi: ApiPromise;
+
+    beforeAll(async function () {
+      paraApi = context.polkadotJs("para");
+      const chainId = (await paraApi.query.ethereumChainId.chainId()).toString();
+      log(`Loading test data for chainId ${chainId}.`);
+    });
+
+    it({
+      id: "C001",
+      title: "Verify MOON-2824",
+      test: async function () {
+        if (paraApi.consts.system.version.specName.toString() !== "moonriver") {
```

Comment:
is this for all `moonbeam` runtimes or for only the Moonbeam network?

If the former then you can use: `chainType: "moonbeam",` on the test case. If the latter then it's `api.rpc.system.chain()`